terraform/spec/dependabot/terraform/requirements_updater_spec.rb

```diff
@@ -85,7 +109,13 @@
         context "that is not satisfied" do
           let(:requirement) { ">= 0.2.1, < 0.3.0" }
           let(:latest_version) { "0.3.7" }
-          its([:requirement]) { is_expected.to eq(">= 0.2.1, < 0.4.0") }
+          its([:requirement]) { is_expected.to eq(">= 0.2.1, < 0.3.7") }
```

Comment:
This requirement excludes the latest version. Is that what you meant to do?

Reply:
No, it should be `< 0.3.8`, good catch.